README.md

```diff
@@ -27,22 +25,69 @@ standard bitcoin data directory directory, but why risk it?
 
 ## Requirements
 
-#### bitcoind
+#### Build a taproot-supporting bitcoind
+
+These workbooks require a `bitcoind` built from the [Optech taproot
+branch, v0.1](https://github.com/bitcoinops/bitcoin/releases/tag/v0.1) which
+supports schnorr and taproot. 
+
+![workshop_repositories](files/0-repositories-diagram.jpg)
```

Comment:
See comment: https://github.com/bitcoinops/taproot-workshop/pull/5#discussion_r318651982

- I think the 'imports' arrow is going the wrong way. Either change the text to 'imported by' or reverse the arrow.
- s/1.0-Workshop-Setup.iypnb/1.0-Workshop-Setup.ipynb/